mmdet/models/dense_heads/ld_head.py

```diff
@@ -0,0 +1,718 @@
+import mmcv
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule, Scale, bias_init_with_prob, normal_init
+from mmcv.runner import force_fp32, load_checkpoint
+
+from mmdet.core import (anchor_inside_flags, bbox2distance, bbox_overlaps,
+                        build_assigner, build_sampler, distance2bbox,
+                        get_classes, images_to_levels, multi_apply,
+                        multiclass_nms, reduce_mean, unmap)
+from mmdet.models import build_detector
+from ..builder import HEADS, build_loss
+from .anchor_head import AnchorHead
+
+
+def init_detector(config, checkpoint=None, device='cuda:0', cfg_options=None):
```

Comment:
This function should not be implemented here. It also seems to duplicate those in the mmdet.apis

Reply:
When I try to import init_detector from mmdet.apis, it tells me "cannot import name 'init_detector' from 'mmdet.apis" . Is there any solution?